src/server/external-services/dummy.ts

```diff
@@ -1,13 +1,18 @@
 const axios = require("axios");
-const { GITHUB_TOKEN } = process.env;
+import dotenv from "dotenv";
+
+dotenv.config();
 
 export class DummyExternalService {
   private GITHUB_API_BASE_URL: string;
+  private GITHUB_TOKEN: string;
 
   constructor() {
     this.GITHUB_API_BASE_URL = "https://api.github.com";
+    this.GITHUB_TOKEN = process.env.GITHUB_TOKEN || "";
   }
 
+
   async getPRFiles(
     owner: string,
     repo: string,
```

Comment:
const axios = require("axios");
-const { GITHUB_TOKEN } = process.env;
+import dotenv from "dotenv";
+
+dotenv.config();
 
 export class DummyExternalService {
   private GITHUB_API_BASE_URL: string;
+  private GITHUB_TOKEN: string;
 
   constructor() {
     this.GITHUB_API_BASE_URL = "https://api.github.com";
+    this.GITHUB_TOKEN = process.env.GITHUB_TOKEN || "";
   }
 
+
   async getPRFiles(
     owner: string,
     repo: string,
hola